### PR TITLE
add meta tags for SEO

### PR DESCRIPTION
**Pull request by fabiosantoscode**
Port of https://github.com/iterative/blog/pull/144

On monday I'll see if all the which should be in here, are.

I've removed some stuff from the blog SEO which was repeated from the DefaultSEO component.

### CODE REVIEW

File: src/components/SEO/index.tsx [hunk 1]
```diff
@@ -72,18 +76,28 @@ const SEO: React.FC<ISEOProps> = ({
...
+        property: 'og:image:width',
+        content: String(image.presentationWidth)
+      },
+      {
+        property: 'og:image:height',
+        content: String(image.presentationHeight)
       },
       {
         name: 'twitter:image',
-        content: encodeURI(`${siteMeta.siteUrl}${image}`)
```
Comment (shcheklein) [inline src/components/SEO/index.tsx:86]: was `encodeURI` actually needed? cc @iAdramelk ?

File: src/templates/blog-post.tsx [hunk 2]
```diff
@@ -67,24 +68,17 @@ interface IBlogPostPageProps {
...
     markdownRemark: IBlogPostData
   }
-  pageContext: {
-    next: IBlogPostData
-    previous: IBlogPostData
-  }
 }
 
-const BlogPostPage: React.FC<IBlogPostPageProps> = ({ data }) => {
+const BlogPostPage: React.SFC<IBlogPostPageProps> = ({ data }) => {
```
Comment (shcheklein) [inline src/templates/blog-post.tsx:73]: SFC is deprecated?

File: src/components/Page/DefaultSEO/index.tsx [hunk 3]
```diff
@@ -76,21 +94,13 @@ const DefaultSEO: React.FC<IDefaultSEOProps> = ({ pageInfo }) => {
       meta={meta}
       link={[
         {
-          rel: 'shortcut icon',
+          rel: 'icon',
           type: 'image/vnd.microsoft.icon',
```
Comment (shcheklein) [inline src/components/Page/DefaultSEO/index.tsx:98]: is this type required? legacy?

File: src/components/Page/DefaultSEO/index.tsx [hunk 4]
```diff
@@ -76,21 +94,13 @@ const DefaultSEO: React.FC<IDefaultSEOProps> = ({ pageInfo }) => {
...
           href: '/favicon.ico'
         },
         {
-          rel: 'icon',
```
Comment (shcheklein) [inline src/components/Page/DefaultSEO/index.tsx:84]: why are we removing these?

File: src/components/SEO/index.tsx (hunk 1 above)
Comment (iAdramelk) [reply src/components/SEO/index.tsx:86]: To be hones I'm not sure. The one who originally added it to the blog was you https://github.com/iterative/blog/commit/3b5c975363fd4f86ef66dd102a2a130759df8fcb . I think then that you where fixing some kind of bug that we have without escaping, but if not, I'm not sure that we need it.

Comment (shcheklein) [reply src/components/SEO/index.tsx:86]: Good question. I see that I was solving a different problem in the first place - absolute URL vs relative one. Don't remember at all why did I put `encodeURI` to be honest.

**Comment by shcheklein** [comment]
Closing this in favor of #1183

**Pull request closed without merging**